.github/actions/caching/action.yml

```diff
@@ -37,40 +34,30 @@ runs:
       cat requirements.txt
     shell: bash
 
-  - run: python -c "import time ; days = time.time() / 60 / 60 / 24 ; print(f'TIME_PERIOD={int(days / ${{ inputs.interval }})}')" >> $GITHUB_ENV
-    if: inputs.requires == 'latest'
-    shell: bash
-
   # Note: This uses an internal pip API and may not always work
   # https://github.com/actions/cache/blob/master/examples.md#multiple-oss-in-a-workflow
   - name: Determine caches
-    env:
-      PWD: $(pwd)
+    id: cache_dirs
     run: |
-      python -c "from pip._internal.locations import USER_CACHE_DIR; print('PIP_CACHE_DIR=' + USER_CACHE_DIR)" >> $GITHUB_ENV
-      python -c "import os ; print('TRANSFORMERS_CACHE=' + os.path.join('$PWD', 'ci-cache_huggingface'))" >> $GITHUB_ENV
+      pip_cache=$(python -c "from pip._internal.locations import USER_CACHE_DIR; print(USER_CACHE_DIR)")
+      echo "pip_cache=$pip_cache" >> $GITHUB_OUTPUT
```

Comment:
nit
```suggestion
      echo "pip_cache=$(pip cache dir)" >> $GITHUB_OUTPUT
```

Reply:
I think there was some issue with Win machines :/ but lets try it again :chipmunk: